paymentsheet/src/test/java/com/stripe/android/paymentelement/embedded/content/DefaultEmbeddedConfigurationHandlerTest.kt

```diff
@@ -53,6 +53,38 @@ internal class DefaultEmbeddedConfigurationHandlerTest {
         assertThat(result.exceptionOrNull()?.message).isEqualTo("Configuring while a sheet is open is not supported.")
     }
 
+    @Test
```

Comment:
You should be able to omit these integration level tests, and test this at the unit level. See `DefaultPaymentElementLoaderTest`.